package.json

```diff
@@ -5,15 +5,19 @@
   "scripts": {
     "pretest": "mvn clean package",
     "test": "karma start angularjs-portal-home/target/web/karma.conf.js  --single-run",
-	"prestart": "mvn clean package",
-	"start": "mvn jetty:run"
+  	"prestart": "mvn clean package",
+  	"start": "mvn jetty:run"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/UW-Madison-DoIT/angularjs-portal.git"
   },
-  "author": "",
-  "license": "ISC",
+  "author": {
+    "name": "University of Wisconsin Madison",
+    "email": "my-admin@lists.wisc.edu",
```

Comment:
Awesome.  I suspect we'll be changing this to an apereo list in the near future, probably the uportal-dev email list.  As of now, it's good.